packages/renderer/src/components/atoms/Percentage/Percentage.tsx

```diff
@@ -0,0 +1,23 @@
+// =============================================================================
+//  Copyright 2022 Haveno
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// =============================================================================
+
+interface PercentageProps {
+  value: number;
+}
+
+export function Percentage({ value }: PercentageProps) {
+  return <>%{value * 100}</>;
```

Comment:
Is this the correct format? Percentage values are supposed to look like `3.45%`, `25%`, etc